Makefile

```diff
@@ -79,6 +79,11 @@ installer-gen: kustomize edit-image
 installer: ## Generates full .yaml output from Kustomize for base and overlays
 	make installer-gen OVERLAY=base && make installer-gen OVERLAY=overlays/kind OVERLAY_LABEL=-kind
 
+.version: ## Uses the git-version-gen script to generate a tag version
+	./git-version-gen > .version
```

Comment:
This wants an arg:
```
$ ./git-version-gen 
Usage: ./git-version-gen [OPTION]... $srcdir/.tarball-version [TAG-NORMALIZATION-SED-SCRIPT]
Print a version string.

Options:

   --prefix PREFIX    prefix of git tags (default 'v')
   --fallback VERSION
                      fallback version to use if "git --version" fails

   --help             display this help and exit
   --version          output version information and exit

Send patches and bug reports to <bug-gnulib@gnu.org>.
```

Reply:
I fixed this to not use any tarball-version files.